app/models/queue_item.rb

```diff
@@ -3,15 +3,30 @@ class QueueItem < ActiveRecord::Base
   belongs_to :video
 
   validates_uniqueness_of :video_id, scope: :user_id
+  validates_uniqueness_of :queue_position, scope: :user_id
+
+  validates_numericality_of :queue_position, { only_integer: true } 
 
   delegate :category, to: :video
   delegate :title, to: :video, prefix: :video
 
   def rating
-    review = Review.where(user_id: user.id, video_id: video.id).first
     review.rating if review
```

Comment:
Hi Matthew.
Thanks for reviewing my code. Would you please explain this to me? Is it better to return an empty string than nil? 
Thanks!Dawn
      From: Matthew Barram <notifications@github.com>
 To: dawnmiceli100/myflix <myflix@noreply.github.com> 
Cc: dawnmiceli100 <dawnmiceli100@yahoo.com> 
 Sent: Tuesday, February 17, 2015 2:22 PM
 Subject: Re: [myflix] Mod 5 (#5)
   
In app/models/queue_item.rb:>  
>    delegate :category, to: :video
>    delegate :title, to: :video, prefix: :video
>  
>    def rating
> -    review = Review.where(user_id: user.id, video_id: video.id).first
>      review.rating if review
Maybe you should return an empty string—
Reply to this email directly or view it on GitHub.